src/java.desktop/share/classes/javax/swing/text/html/HTMLDocument.java

```diff
@@ -24,16 +24,43 @@
  */
 package javax.swing.text.html;
 
+import sun.swing.SwingUtilities2;
```

Comment:
Hmm .. there's no change in this file except for import clean up.
Ordinarily I'd say revert but I can see it was quite a chunk ..

Reply:
update .. please revert .. I guess the IDE can re-do this when we touch it next